main.go

```diff
@@ -326,7 +324,7 @@ func (o *searchRepoOptions) buildIndex(out io.Writer) (*search.Index, error) {
 		ind, err := repo.LoadIndexFile(f)
 		if err != nil {
 			// TODO should print to stderr
```

Comment:
Can you please remove the `TODO` since you've done this.

Thanks for your PR :+1: